src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -263,6 +263,7 @@ div.sf-toolbar .sf-toolbar-block a:hover {
     border-width: 0;
     position: relative;
     top: 8px;
+    vertical-align: baseline;
```

Comment:
should it actually also apply to images ?

Reply:
For me, the css rule already exist in `.sf-toolbarreset *` selector. It's not necessary to duplicate this rule. Just improve it.

Add `!important` and that's ok. No ?